### PR TITLE
The markup of the advertising banner is broken on http://www.rediff.com (close #387)

### LINKED ISSUE #387: The markup of the advertising banner is broken on http://www.rediff.com
image

Code that create banner:
```javascript
__call$(document, "write", ['<script language="javascript" type="text/javascript">\n', "__begin$"]);
__call$(document, "write", ['if(document.getElementById("hm_rht_adcontainer"))\n']);
__call$(document, "write", ['{\n']);
__call$(document, "write", ['document.getElementById("hm_rht_adcontainer").style.display = "block";\n']);
__call$(document, "write", ['}\n']);
__call$(document, "write", ['</script>\n']);
__call$(document, "write", ['<html>\n']);
__call$(document, "write", [' <body>\n']);
__call$(document, "write", ['  <table border="0" cellspacing="0" cellpadding="0" width="300" height="250">\n']);
__call$(document, "write", ['	<tr>\n']);
__call$(document, "write", ['		<td><a href="http://ads.rediff.com/RealMedia/ads/click_lx.ads/us.rediff.com/mainhome.htm/L-26/96003601/Right/OasDefault/Shop_IA_HP_JFM16/Shop_IA_HP_OND15_04.html/7649416b554661764d61594143552f76?URL\\']);
__call$(document, "write", ['" target="_blank"><img src="http://imads.rediff.com/0/OasDefault/Shop_IA_HP_JFM16/usa_300x250_IAHP_lakme.jpg" border="0"></a></td>\n']);
__call$(document, "write", ['	</tr>\n']);
__call$(document, "write", ['</table>\n']);
__call$(document, "write", [' </body>\n']);
__call$(document, "write", ['</html>\n', "__end$"]);
```

### PR DESCRIPTION
**Pull request by LavrovArtem**
\cc @churkin @AndreyBelym
@VasilyStrelyaev please check test name

### CODE REVIEW

File: test/client/fixtures/sandbox/node/document-test.js [hunk 1]
```diff
@@ -435,3 +435,26 @@ asyncTest('document elements are overridden after document.write has been called
 
     document.body.appendChild(iframe);
 });
+
+asyncTest('multiple document.write with html and body tags should not broke markup (GH-387)', function () {
```
Comment (inikulin) [inline test/client/fixtures/sandbox/node/document-test.js]: `should not brake markup`

File: test/client/data/node-sandbox/write-advertising-banner.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
```
Comment (inikulin) [inline test/client/data/node-sandbox/write-advertising-banner.html]: `write-advertising-banner.html` - this name is strange =)

File: test/client/fixtures/sandbox/node/document-test.js (hunk 1 above)
Comment (VasilyStrelyaev) [inline test/client/fixtures/sandbox/node/document-test.js]: +1

**Comment by VasilyStrelyaev** [comment]
\r-

**Comment by LavrovArtem** [comment]
FPR

File: test/client/fixtures/sandbox/node/document-test.js [hunk 3]
```diff
@@ -435,3 +435,26 @@ asyncTest('document elements are overridden after document.write has been called
 
     document.body.appendChild(iframe);
 });
+
+asyncTest('multiple document.write with html and body tags should not brake markup (GH-387)', function () {
```
Comment (VasilyStrelyaev) [inline test/client/fixtures/sandbox/node/document-test.js]: No-no!
*Break* of course!
```
asyncTest('multiple document.write with html and body tags should not break markup (GH-387)', function () {
```

Comment (VasilyStrelyaev) [inline test/client/fixtures/sandbox/node/document-test.js]: I got confused a bit

**Comment by VasilyStrelyaev** [comment]
\r-

**Comment by LavrovArtem** [comment]
FPR

**Comment by VasilyStrelyaev** [comment]
lgtm

**Comment by inikulin** [comment]
lgtm

**Comment by churkin** [comment]
lgtm

**Pull request merged**